docs/docs/collections/lists.md

```diff
@@ -312,4 +312,15 @@ Note: The first item to satisfy the callback is returned.
 print([1, 2, 3].findIndex(def (item) => item == 2)); // 1
 print([1, 2, 3, 4, 5, 6].findIndex(def (item) => item % 2 == 0, 2)); // 3
 print([1, 2, 3, 4, 5, 6].findIndex(def (item) => item % 2 == 0, 2, 3)); // nil
-```
\ No newline at end of file
+```
+
+### list.Splice(Number, Number, List) -> List
```

Comment:
```suggestion
### list.splice(Number, Number, List) -> List
```

I think it would be good to have a dedicated note about the return of `.splice()`. Comparing it to JS splice modifies in place and it's return value is an array containing all removed elements - this is different to what is happening here with it essentially making a shallow copy and returning the modified array rather than mutating in place